faq.md

```diff
@@ -288,18 +287,12 @@ See our [roadmap](roadmap.html).
 
 ## Can I use Bazel for my [INSERT LANGUAGE HERE] project?
 
-We have an extension mechanism called Skylark that allows you to add new rules
-without recompiling Bazel.
+Bazel is extensible. Anyone can add support for new languages. Many languages
+are supported, see the [build encyclopedia](/docs/be/overview.html) for a list
```

Comment:
/docs/be/overview.html#additional-rules

Will point them right at the additional rules section where the languages are listed.

Reply:
Done